ext/http/http_next.rs

```diff
@@ -382,11 +385,11 @@ pub fn op_http_read_request_body(
   state.resource_table.add_rc(body_resource)
 }
 
-#[op(fast)]
+#[op2]
 pub fn op_http_set_response_header(
-  slab_id: SlabId,
-  name: ByteString,
-  value: ByteString,
+  #[smi] slab_id: SlabId,
+  #[serde] name: ByteString,
```

Comment:
We need to add `fast` `ByteString` support for `#[string]` parameters.

Reply:
That would be lovely

Reply:
https://github.com/denoland/deno_core/issues/108